docs/source/notebooks/mmm/mmm_build_from_yml_example.ipynb

```diff
@@ -0,0 +1,1891 @@
+{
```

Comment:
<p>Can you !cat the file to show the content and explain the format a bit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>